game/permanent.go

```diff
@@ -17,11 +17,12 @@ type Permanent struct {
 	Id PermanentId
 
 	// Properties that are relevant for any permanent
-	Auras      []*Permanent
-	Effects    []*Effect
-	Owner      *Player
-	Tapped     bool
-	TurnPlayed int
+	Auras       []*Permanent
+	DidActivate bool
```

Comment:
this makes it sound like this bool is whether the ability ever activated. but i'm guessing you want it to mean, whether this ability was activated this turn? maybe call it ActivatedThisTurn?